src/plugins/credential_management/public/components/credential_table/credentials_table.tsx

```diff
@@ -75,6 +79,10 @@ export const CredentialsTable = ({ canSave, history }: Props) => {
     setBreadcrumbs(getListBreadcrumbs());
   });
 
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [isDelLoading, setIsDelLoading] = React.useState(false);
+  const [delModalVisible, setDelModalVisible] = React.useState(false);
```

Comment:
delModalVisible -> confirmDelete? Let's also refer to variable naming in other plugins